datadog_lambda/wrapper.py

```diff
@@ -339,6 +347,9 @@ def _after(self, event, context):
                     self.span.set_tag("http.status_code", status_code)
                 self.span.finish()
 
+            if llmobs_env_var:
+                LLMObs.flush()
```

Comment:
I am guessing that this flush should be moved until after all spans have been created. We create the inferred spans and cold start spans below. Probably best to place it with the other flush calls below.

Reply:
Good point! I'l make the change and verify it works in my test lambda function.